tasties_app/templates/tasties_app/single_recipe.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'tasties_app/base.html' %}
+{% load static %}
+{% block css %}
+    {% load bootstrap_icons %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/single_recipe.css' %}" />
+{% endblock %}
+{% block content %}
+    <div class="recipe-body">
+        <div class="recipe-header">
+            <h1>{{ recipe.title }}</h1>
+            <h2>{{ recipe.description }}</h2>
```

Comment:
You know what, I would also move the description lower down (between the recipe picture and the directions), and also not make it an h2, as that might be too bold (but it might be the right choice to leave it as an h2 because of accessibility concerns, just modify the font-weight for h2 in the CSS). See [this](https://www.epicurious.com/recipes/food/views/air-fryer-brussels-sprouts-salad-grain-bowl#:~:text=It%20might%20be,if%20you%20like.) as an example that descriptions can be a bit verbose.